event-based-simulators/main/cumulocityAPI.py

```diff
@@ -18,15 +16,8 @@
     'Authorization': 'Basic ' + user_and_pass
 }
 
-# JSON-PYTHON mapping, to get json.load() working
-null = None
-false = False
-true = True
-######################
 log = logging.getLogger("C8yAPI")
 
-OEE_DATA_MODEL_FIELD_NAME = "@com_adamos_oee_datamodel_MachineOEEConfiguration"
```

Comment:
Iam not sure wether this line is really needed or not, just vsc couldn't find die beeing referenced anywhere

Reply:
I dont think it is required